lib/api/element-commands/isVisible.js

```diff
@@ -60,7 +61,7 @@ class IsVisible extends BaseElementCommand {
       browserVersion = parseInt(browserVersion, 10);
 
       // Are we running 12 or newer?
-      if (browserName.toLowerCase() === 'safari' && (browserVersion >= 12) && this.settings.webdriver.start_process === true) {
+      if (browserName.toLowerCase() === BrowserName.SAFARI && (browserVersion >= 12) && this.settings.webdriver.start_process === true) {
         return this.executeProtocolAction('getElementProperty', ['hidden']).then(result => ({
           value: result.value === false
         }));
```

Comment:
Same thing down here.
```suggestion
        return this.executeProtocolAction('getElementProperty', ['hidden']).then(result => ({
          value: result.value === false,
          status: result.status
        }));
```